Backpack-SwiftUI/Skeleton/Classes/BPKSkeleton.swift

```diff
@@ -0,0 +1,103 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import Backpack_Common
+
+public struct BPKSkeleton: View {
+    private var type: BPKSkeleton.ViewType = .image
+    private var size: BPKSkeleton.Size = .default
+    private var style: BPKSkeleton.Style = .default
+    private var color: Color = Color(.skyGrayTint06.darkVariant(.blackTint02))
+
+    @State private var phase: CGFloat = 0
+    let duration = 1.0
+    let delay = 0.2
+    
+    public init(
```

Comment:
Could we make this init private? This will probably 'force' consumers to use the static functions in the extension.

(Please also add a comment that says this above the init)
"Please use the static functions to get the preferred skeleton" (or something along those lines)